docs/source/testing.rst

```diff
@@ -97,41 +97,112 @@ Smoke test:
     $ cd koku/testing
     $ ./run_smoke_tests.sh
     
+=====================
+PDB in Docker
+=====================
```

Comment:
So, the extra `=` here are causing the the title bold syntax to bleed into the line below.  The `=` should only follow up to the end of the text. 

Fox example,
```
=============
PDB in Docker
=============
```